components/style/servo_config_facade.rs

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ /* Ref: https://github.com/servo/servo/issues/15460 for more info for why it's needed. */
```

Comment:
I think adding a brief explanation about this module with `//!` comments would be better instead.